package.json

```diff
@@ -31,9 +31,8 @@
     "chromatic": "npx chromatic storybook:build"
   },
   "peerDependencies": {
-    "prop-types": "^15.7.2",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
     "react-router-dom": "^5.2.0",
```

Comment:
should the reaxt-router-dom stil be here?

Reply:
Yep it's still peer dependency, but now you don't need to have it installed

Reply:
ok